src/main/java/money/Account.java

```diff
@@ -0,0 +1,263 @@
+package money;
+
+import controlpanel.DukeException;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.Comparator;
+
+public class Account {
+    private ArrayList<Income> incomeListTotal;
+    private ArrayList<Expenditure> expListTotal;
+    private ArrayList<Income> incomeListCurrMonth;
+    private ArrayList<Expenditure> expListCurrMonth;
+    private ArrayList<Goal> shortTermGoals;
+    private ArrayList<Instalment> instalments;
+    private ArrayList<BankTracker> bankTrackerList;
+    private ArrayList<Loan> loans;
+    private float totalSavings;
+    private float currMonthSavings;
+    private float baseSavings;
+    private float goalSavings;
+    private boolean toInitialize;
+
+    public Account() {
+        incomeListTotal = new ArrayList<>();
+        expListTotal = new ArrayList<>();
+        incomeListCurrMonth = new ArrayList<>();
+        expListCurrMonth = new ArrayList<>();
+        shortTermGoals = new ArrayList<>();
+        instalments = new ArrayList<>();
+        bankTrackerList = new ArrayList<>();
+        loans = new ArrayList<>();
+        toInitialize = true;
+    }
+
+    public Account(Account account) {
+        incomeListTotal = account.getIncomeListTotal();
+        expListTotal = account.getExpListTotal();
+        incomeListCurrMonth = account.getIncomeListCurrMonth();
+        expListCurrMonth = account.getExpListCurrMonth();
+        shortTermGoals = account.getShortTermGoals();
+        instalments = account.getInstalments();
+        bankTrackerList = account.getBankTrackerList();
+        loans = account.getLoans();
+        toInitialize = account.isToInitialize();
+        baseSavings = account.getBaseSavings();
+        updateSavings();
+        populateCurrentMonthLists();
+//        if (account.isInitialised()) {
+//            toInitialize = false;
+//        } else { toInitialize = true; }
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.